packages/components/src/higher-order/with-draggable/index.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * WordPress Dependencies
+ */
+import { Component } from '@wordpress/element';
+import { createHigherOrderComponent } from '@wordpress/compose';
+
+const dragImageClass = 'components-draggable__invisible-drag-image';
+const cloneWrapperClass = 'components-draggable__clone';
+const cloneHeightTransformationBreakpoint = 700;
+const clonePadding = 20;
+
+const withDraggable = createHigherOrderComponent(
```

Comment:
Note that this code has been ported with no substantial modifications from the `Draggable` component.